### ISSUE TITLE
[Question] How to apply ITS Theme Alternate Styling & smaller newbie questions

### ISSUE THREAD

**Issue by Hyfro-Cyrup** [oleeskild/obsidian-digital-garden#319]
I use the ITS theme with alternate styling for all of my notes. I uploaded the full css snippet to github, and I was able to inspect element and confirm that the classes defined therein do exist, but they're not being applied globally like they should be given my Style Settings plugin options. I'll add in screenshots of what it looks like now with just the base ITS theme applied and something closer to what it should look like with the correct class shoved in there in a div. 
![image](https://github.com/oleeskild/obsidian-digital-garden/assets/108480855/892ac848-5dfb-47b3-bb50-c57e3754b28a)
Current
![image](https://github.com/oleeskild/obsidian-digital-garden/assets/108480855/be708e29-b619-426b-94fb-233c155efbf5)
Closer to desired

Semi-related questions I have as someone who has never done something like this before are:
- Is there something extra I should be doing to make the note title show up? I already checked the box under "Global Note Settings".
- Any inline dataview query gives an error message about updating my dataview plugin and causes the entire note to not be uploaded. I saw other issues with a back and forth relationship between Digital Garden and inline Dataview, so I was curious where we're sitting on that currently.

**Comment by rokville** [comment]
(semi off-topic) Hyfro-Cyrup, I am using my DG for what looks like the same purpose as you are... D&D notes! I'd love to know how you are organizing and presenting and maybe follow your progress, if you were open to sharing.

**Comment by GJRobert** [comment]
@Hyfro-Cyrup Hi, are you using ITS theme in your local Obsidian vault now too? (making sure)

**Comment by GJRobert** [comment]
@Hyfro-Cyrup Hi, are you using ITS theme in your local Obsidian vault now too? (making sure)

**Comment by Hyfro-Cyrup** [comment]
@GJRobert sorry I didn't see your replies until now. Yes, I am using the ITS theme in my local vault. 
I did manage to find a solution/workaround by appending the themes I want to use in two places: 
1. .env line 2 now reads `BASE_THEME=dark dnd t-d` (I added "dnd t-d")
2. meta.js line 67 now reads `    baseTheme: process.env.BASE_THEME || "dark dnd t-d",` (I added "dnd t-d")

And I appreciate your interest in my notes, @rokville! Feel free to email me and I'll send you a link to my garden.
